internal/pkg/cli/deploy/errors.go

```diff
@@ -38,3 +40,38 @@ func (e *errSvcWithALBAliasHostedZoneWithCDNEnabled) RecommendActions() string {
 	}
 	return strings.Join(msgs, "\n")
 }
+
+type errEnvHasPublicServicesWithRedirect struct {
+	services []string
+}
+
+func (e *errEnvHasPublicServicesWithRedirect) Error() string {
```

Comment:
I still don't know if it's common for an `Error()` to return a multi-line string that's colored. 
Do you know of any multi-line string in the `stdlib`?

My recommendation is to keep `Error` to be a succinct single line message like:
```
Error() {
   "{len(services) redirect HTTP to HTTPS"
}
```
and have this nice explanation in `RecommendedActions`.

Reply:
that makes sense - see how it looks now!

Reply:
Looks great! ty.

nit: doc-comments are missing